.travis.yml

```diff
@@ -38,7 +38,7 @@ matrix:
     script:
     - cargo check --tests
   - env: CLIPPY
-    rust: 1.31.0  # `stable`: Locking down for consistent behavior
+    rust: 1.31.0
```

Comment:
btw why is the comment removed?

Reply:
uh, I don't remember doing that ^^' I'll restore it